src/theme/dialog/holder.js

```diff
@@ -18,8 +17,10 @@ const DialogHolder = ({ history }) => {
   }, [history.location.search]);
 
   const activeToken = useSelector(selectActiveToken);
-  const networkIsSet = useSelector(state => !!state.network.name);
-  const account = useSelector(selectActiveTokenAccount);
+  const networkIsSet = useSelector((state) => !!state.network.name);
+  const wallet = useSelector(selectActiveTokenAccount);
+  const [account] = useCurrentAccount();
+  const isAuthenticated = Object.keys(account).length > 0 || !!wallet.summary;
```

Comment:
selectActiveTokenAccount is deprecated now
```suggestion
  const [account] = useCurrentAccount();
  const isAuthenticated = Object.keys(account).length > 0;
```